rosbag2_cpp/src/rosbag2_cpp/service_utils.cpp

```diff
@@ -31,6 +31,11 @@ bool is_service_event_topic(const std::string & topic_name, const std::string &
   if (topic_name.length() <= kServiceEventTopicPostfixLen) {
     return false;
   } else {
+    // Not services for action
+    if (topic_name.find("/_action/") != std::string::npos) {
+      return false;
+    }
+
```

Comment:
Please rewrite this if-else block for the sake of runtime performance and clarity in logic as 
```cpp
if (topic_name.length() <= kServiceEventTopicPostfixLen) {
    return false;
  } else if (topic_name.substr(topic_name.length() - kServiceEventTopicPostfixLen) !=
    RCL_SERVICE_INTROSPECTION_TOPIC_POSTFIX)
  {
    // The end of the topic name should be "/_service_event"
    return false;
  } else if (topic_name.find("/_action/") != std::string::npos) {
    // Exclude services for action
    return false;
  }
```
i.e. use check for the services for action at the end of the if-else to keep a similar runtime performance as before.